### PR TITLE
removing unused constants

### PR DESCRIPTION
**Pull request by albertshau**
Removed yarn.user from cdap-default.xml and from Constants.java
since it is unused in the code. Also tried setting it in the conf
to verify that it doesn't do anything.
Other than that, all other constants are unused in the code and
not in cdap-default.xml.

### CODE REVIEW

**Comment by albertshau** [comment]
http://builds.cask.co/browse/CDAP-RBT553

File: cdap-common/src/main/resources/cdap-default.xml [hunk 1]
```diff
@@ -136,14 +136,6 @@
   </property>
 
   <property>
-    <name>yarn.user</name>
```
Comment (sreevatsanraman) [inline cdap-common/src/main/resources/cdap-default.xml]: Might want to check if there are references in docs for this constant and remove those as well.

**Comment by sreevatsanraman** [comment]
Minor comment about docs. LGTM otherwise.

**Comment by albertshau** [comment]
the only reference seems to be in the cdap-site.xml addendum, which is generated from cdap-default.xml.  Built docs locally and verified that the row is removed.  It does complain about some md5 hash though so I'll update that.

**Comment by wolf31o2** [comment]
:+1: LGTM...

**Pull request merged**